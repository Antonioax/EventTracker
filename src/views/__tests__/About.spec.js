import { mount } from "@vue/test-utils";
import { beforeEach, describe, expect, it } from "vitest";

import About from "@/views/About.vue";

describe("About.vue", () => {
  let wrapper;

  beforeEach(() => {
    wrapper = mount(About, {});
  })

  it("should mount component", () => {
    expect(wrapper).toBeTruthy();
  });

  it("should render text", () => {
    expect(wrapper.text()).toContain("This is an about page.");
  });

  it("should match snapshot", () => {
    expect(wrapper.html()).toMatchSnapshot();
  })
});
