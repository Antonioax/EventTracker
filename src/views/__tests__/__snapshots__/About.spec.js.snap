// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`About.vue > should match snapshot 1`] = `
"<div class="about">
  <h1>This is an about page.</h1>
</div>"
`;
